client-spark/spark-2/src/test/scala/org/apache/spark/shuffle/celeborn/CelebornShuffleManagerSuite.scala

```diff
@@ -49,7 +49,7 @@ class SparkShuffleManagerSuite extends Logging {
   }
 
   @junit.Test
-  def testClusterNotAvailable(): Unit = {
+  def testClusterNotAvailableWithQuota(): Unit = {
```

Comment:
Given that we haven't initiated any Celeborn server instances, the current test case only addresses the `Cluster Not Available` scenario, not the `Cluster Not Available With Quota` scenario.

Reply:
+1, good catch! We should leave `SparkShuffleManagerSuite` as it is, and add integration test in stead in `spark-it` module. You can specify worker numbers in `setUpMiniCluster`, see `CelebornHashCheckDiskSuite`.

Reply:
@cfmcgrady, @waitinfuture, IMO, `CelebornHashCheckDiskSuite` includes the verification whether cluster has available workers. Threrefore, it doesn't need to add extra test cases to verify.